packages/react/src/components/Table/TableHead/FilterHeaderRow/FilterHeaderRow.jsx

```diff
@@ -204,11 +204,12 @@ class FilterHeaderRow extends Component {
                 <div />
               ) : column.options ? (
                 column.isMultiselect ? (
-                  <MultiSelect
+                  <MultiSelect.Filterable
                     key={columnStateValue}
                     className={`${iotPrefix}--filterheader-multiselect`}
                     id={`column-${i}`}
                     aria-label={filterText}
+                    placeholder={column.placeholderText || 'Choose an option'}
```

Comment:
we should this with  all our i18n strings moving forward. It would save us from having to define big objets that muddy up the component and we wouldn't need to merge default with props.